## About

 * Applying metagenomics & machine learning to study metabolism in human gut microbiomes ğŸ’©
 * Open source, transparent, and reproducible research + software enthusiast ğŸ¦¾
 * Preferred programming languages: Bash, R, Python, MATLAB ğŸ’»
 * High performance computer cluster aficionado ğŸ›©ï¸ <img align="right" src="https://user-images.githubusercontent.com/35606471/117464909-08338680-af49-11eb-8b36-b4b7cd30a3a3.jpg" height = 200 width = 160>
 * Spoken languages: English, Spanish, Italian ğŸ’¬
 * Dual citizen from Argentina ğŸ‡¦ğŸ‡· / Italy ğŸ‡®ğŸ‡¹

## Education

 * Ph.D. Biology - University of Cambridge ğŸ‡¬ğŸ‡§ 2020 - Present
 * M.Sc. Biotechnology - Chalmers University of Technology ğŸ‡¸ğŸ‡ª 2017 - 2019
 * B.Sc. Biological Systems Engineering - University of California, Davis ğŸ‡ºğŸ‡¸ 2013 - 2017

## Work Experience

 * Computational biologist - European Molecular Biology Laboratory, Heidelberg ğŸ‡©ğŸ‡ª 2019 - 2020

## Software

* ğŸ’ `metaGEM` Snakemake workflow for reconstruction & simulation of genome scale metabolic models directly from metagenomic data 
[![GitHub](https://img.shields.io/badge/GitHub-metaGEM-blue)](https://github.com/franciscozorrilla/metaGEM)

* ğŸ“Š `com-dFBA` MATLAB implementation of community dynamic flux balance analysis (dFBA) for iGEM 2018 
   [![GitHub](https://img.shields.io/badge/GitHub-com--dFBA-yellowgreen)](https://github.com/franciscozorrilla/Chalmers-iGEM2018)

## Publications

* [![DOI](https://img.shields.io/badge/bioRxiv-10.1101%2F2020.12.31.424982%20-B31B1B)](https://www.biorxiv.org/content/10.1101/2020.12.31.424982v2.full)
```
metaGEM: reconstruction of genome scale metabolic models directly from metagenomes
Francisco Zorrilla, Kiran R. Patil, Aleksej Zelezniak
bioRxiv 2020.12.31.424982; doi: https://doi.org/10.1101/2020.12.31.424982 
```

* [![DOI](https://img.shields.io/badge/bioRxiv-10.1101%2F2020.12.31.424982%20-B31B1B)](https://www.biorxiv.org/content/10.1101/2021.06.18.448943v1.full)
```
Reconstruction of genome--scale metabolic model for Hansenula polymorpha using RAVEN toolbox
Francisco Zorrilla, Eduard J Kerkhoven
bioRxiv 2021.06.18.448943; doi: https://doi.org/10.1101/2021.06.18.448943 
```

 * [![arxiv](https://img.shields.io/badge/bioRxiv-10.1101%2F2020.12.13.422558%20-B31B1B)](https://www.biorxiv.org/content/10.1101/2020.12.13.422558v2.full)
```
Plastic-degrading potential across the global microbiome correlates with recent pollution trends
Jan Zrimec, Mariia Kokina, Sara Jonasson, Francisco Zorrilla, Aleksej Zelezniak
bioRxiv 2020.12.13.422558; doi: https://doi.org/10.1101/2020.12.13.422558 
```

 * [![nature](https://img.shields.io/badge/Nature-communications%20biology-darkgreen)](https://www.nature.com/articles/s42003-020-01127-5) (see `Chalmers-Gothenburg` under the author `iGEM Interlab Study Contributors`)
```
Robust estimation of bacterial cell count from optical density
Jacob Beal, Natalie G. Farny, Traci Haddock-Angelli, Vinoo Selvarajah, Geoff S. Baldwin, Russell Buckley-Taylor, Markus Gershater, Daisuke Kiga, John Marken, Vishal Sanchania, Abigail Sison, Christopher T. Workman & iGEM Interlab Study Contributors
Nature communications biology s42003-020-01127-5; doi: https://doi.org/10.1038/s42003-020-01127-5
``` 

## Contact

[![LinkedIn](https://img.shields.io/badge/LinkedIn-fzorrilla94-blue)](https://www.linkedin.com/in/fzorrilla94/)
[![scholar](https://img.shields.io/badge/Google-Scholar-%230492C2)](https://scholar.google.com/citations?user=byUq0i4AAAAJ&hl=en)
[![GitHub](https://img.shields.io/badge/GitHub-franciscozorrilla-9cf)](https://github.com/franciscozorrilla)
[![Twitter](https://img.shields.io/badge/Twitter-%40metagenomez-lightblue)](https://twitter.com/metagenomez)
[![email](https://img.shields.io/badge/email-fz274%40cam.ac.uk-%23a6bddb)](fz274@cam.ac.uk)
