## Overview

 * Omics-driven metabolic modeling & machine learning to study microbial communities 💩
 * Open source, transparent, and reproducible research + software 🦾
 * Programming languages: Bash, R, Python, MATLAB 💻
 * High performance computer cluster aficionado 🛩️ <img align="right" src="https://user-images.githubusercontent.com/35606471/117464909-08338680-af49-11eb-8b36-b4b7cd30a3a3.jpg" height = 200 width = 160>
 * Spoken languages: English, Spanish 💬
 * Citizenship: Argentina 🇦🇷 / Italy 🇮🇹

## Education

 * Ph.D. Biology - University of Cambridge 🇬🇧 2020 - 2024
 * M.Sc. Biotechnology - Chalmers University of Technology 🇸🇪 2017 - 2019
 * B.Sc. Biological Systems Engineering - University of California, Davis 🇺🇸 2013 - 2017

## Work Experience

 * Computational biologist - European Molecular Biology Laboratory, Heidelberg 🇩🇪 2019 - 2020

## Software

[![GitHub](https://img.shields.io/badge/GitHub-metaGEM-blue)](https://github.com/franciscozorrilla/metaGEM)

* 💎 `metaGEM` is a workflow that enables the reconstruction & simulation of genome scale metabolic models directly from metagenomic data 

[![GitHub](https://img.shields.io/badge/GitHub-com--dFBA-yellowgreen)](https://github.com/franciscozorrilla/Chalmers-iGEM2018)

* 📊 `com-dFBA` is a MATLAB implementation of community dynamic flux balance analysis (dFBA), developed for iGEM 2018

## Publications

```
Robust estimation of bacterial cell count from optical density
Jacob Beal, Natalie G. Farny, Traci Haddock-Angelli, Vinoo Selvarajah, Geoff S. Baldwin, Russell Buckley-Taylor, Markus Gershater, Daisuke Kiga, John Marken, Vishal Sanchania, Abigail Sison, Christopher T. Workman & iGEM Interlab Study Contributors
Nature communications biology, 2020; https://doi.org/10.1038/s42003-020-01127-5
``` 

```
Comparative analysis of three studies measuring fluorescence from engineered bacterial genetic constructs
Jacob Beal, Geoff S Baldwin, Natalie G Farny, Markus Gershater, Traci Haddock-Angelli, Russell Buckley-Taylor, Ari Dwijayanti, Daisuke Kiga, Meagan Lizarazo, John Marken, Kim de Mora, Randy Rettberg, Vishal Sanchania, Vinoo Selvarajah, Abigail Sison, Marko Storch, Christopher T Workman, iGEM Interlab Study Contributors
PloS one, 2021; https://doi.org/10.1371/journal.pone.0252263
```

```
metaGEM: reconstruction of genome scale metabolic models directly from metagenomes
Francisco Zorrilla, Filip Buric, Kiran R Patil, Aleksej Zelezniak
Nucleic acids research, 2021; https://doi.org/10.1093/nar/gkab815
``` 

```
Plastic-degrading potential across the global microbiome correlates with recent pollution trends
Jan Zrimec, Mariia Kokina, Sara Jonasson, Francisco Zorrilla, Aleksej Zelezniak
MBio, 2021; https://doi.org/10.1128/mBio.02155-21
```

```
Microbial communities form rich extracellular metabolomes that foster metabolic interactions and promote drug tolerance
Jason SL Yu, Clara Correia-Melo, Francisco Zorrilla, Lucia Herrera-Dominguez, Mary Y Wu, Johannes Hartl, Kate Campbell, Sonja Blasche, Marco Kreidl, Anna-Sophia Egger, Christoph B Messner, Vadim Demichev, Anja Freiwald, Michael Mülleder, Michael Howell, Judith Berman, Kiran R Patil, Mohammad Tauqeer Alam, Markus Ralser
Nature microbiology, 2022; https://doi.org/10.1038/s41564-022-01072-5
```

```
Reconstruction of genome--scale metabolic model for Hansenula polymorpha using RAVEN toolbox
Francisco Zorrilla, Eduard J Kerkhoven
Yeast Metabolic Engineering, 2022 ; https://doi.org/10.1101/2021.06.18.448943 
```

## Contact

[![LinkedIn](https://img.shields.io/badge/LinkedIn-fzorrilla94-blue)](https://www.linkedin.com/in/fzorrilla94/)
[![scholar](https://img.shields.io/badge/Google_Scholar-Francisco_Zorrilla-%230492C2)](https://scholar.google.com/citations?user=byUq0i4AAAAJ&hl=en)
[![GitHub](https://img.shields.io/badge/GitHub-franciscozorrilla-9cf)](https://github.com/franciscozorrilla)
[![Twitter](https://img.shields.io/badge/Twitter-%40metagenomez-lightblue)](https://twitter.com/metagenomez)
[![email](https://img.shields.io/badge/email-fz274%40cam.ac.uk-%23a6bddb)](fz274@cam.ac.uk)

*Last updated 28/02/2023*
